src/libs/antares/study/parameters.cpp

```diff
@@ -1045,26 +1100,7 @@ bool Parameters::loadFromINI(const IniFile& ini, uint version, const StudyLoadOp
     // forcing value
     if (options.nbYears != 0)
     {
-        if (options.nbYears > nbYears)
-        {
-            if (yearsFilter)
-            {
-                // The variable `yearsFilter` must be enlarged
-                bool* newset = new bool[options.nbYears];
-                for (uint i = 0; i != nbYears; ++i)
-                    newset[i] = yearsFilter[i];
-                for (uint i = nbYears; i < options.nbYears; ++i)
-                    newset[i] = false;
-                delete[] yearsFilter;
-                yearsFilter = newset;
-            }
-            else
-            {
-                yearsFilter = new bool[options.nbYears];
-                for (uint i = 0; i < options.nbYears; ++i)
-                    yearsFilter[i] = true;
```

Comment:
If there was no yearsFilter we init filter with true.

With new implementation resize is done with false value.

To be discussed.

Reply:
done